enrichments/trace/internal/elastic/span.go

```diff
@@ -38,6 +38,12 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+// Here we are duplicating newer semconv attributes because since v1.26.0
```

Comment:
is this comment relevant to this file? specifically the duplicating symbols part

Reply:
I would prefer upgrading the semconv to the latest version. If there are any breaking changes in attribute names, we would want to support the updated names, but, even if you don't want to do that I think we can import 2 semconv versions for now. Something like:

```
	semconv25 "go.opentelemetry.io/collector/semconv/v1.25.0"
	semconv26 "go.opentelemetry.io/collector/semconv/v1.26.0"
```

This will make it easier to find which semconv is supported and avoid confusion related to supporting non-semconv fields.

Reply:
@lahsivjar the `NetXXX` attributes we are using here are gone, there are the renamed one `NetworkXXX`. I guess the code should be updated to handle both?

Reply:
>  I guess the code should be updated to handle both?

Yes, but that would be outside the scope of this PR. We also don't have a policy on what the support matrix would look like in such cases (like how much older semconv we would need to support).

For now, how about importing both the semconv versions?

Reply:
> https://github.com/elastic/opentelemetry-lib/issues/129

I have created an issue for supporting the newer semconv.

Reply:
@lahsivjar do you want me to already switch the attributes available in 1.27.0 to semconv27 so it's clear what needs to be updated?

Reply:
> do you want me to already switch the attributes available in 1.27.0 to semconv27 so it's clear what needs to be updated

I think it would be better as a separate PR.